### PR TITLE
cnf-tests: Fix fec secure boot test case

### PR DESCRIPTION
**Pull request by kononovn**
Adding dynamic secureBoot verification.
If secure boot enabled SriovFecClusterConfig resources gets updated with the correct pfDriver configuration vfio-pci

### CODE REVIEW

File: cnf-tests/testsuites/e2esuite/fec/fec.go [hunk 1]
```diff
@@ -66,32 +67,49 @@ var _ = Describe("[fec]", func() {
...
 	}
 
 	pci, err := getAcc100PciFromNode(nn[0])
 	if err != nil {
-		return "", "", err
+		return "", "", false, err
+	}
+
+	// Get secure boot status from node.
+	acc100Node, err := client.Client.Nodes().Get(context.TODO(), nn[0], metav1.GetOptions{})
```
Comment (zeeke) [inline cnf-tests/testsuites/e2esuite/fec/fec.go:87]: please, consider move this code block to a `isSecureBoot(nodeName string) (bool, error)` function and call it where it's needed.

File: cnf-tests/testsuites/e2esuite/fec/fec.go [hunk 2]
```diff
@@ -43,18 +44,18 @@ var _ = Describe("[fec]", func() {
...
 		Expect(err).ToNot(HaveOccurred())
 
-		err = createSriovFecClusterObject(nodeName, pciAddress, numVfs, isSingleNode)
+		err = createSriovFecClusterObject(nodeName, pciAddress, numVfs, isSingleNode, isSecureBootEnabled)
 		Expect(err).ToNot(HaveOccurred())
 	})
 
 	Context("Expose resource on the node", func() {
-		It("should show resources under the node", func() {
+		FIt("should show resources under the node", func() {
```
Comment (zeeke) [inline cnf-tests/testsuites/e2esuite/fec/fec.go:58]:
```suggestion
		It("should show resources under the node", func() {
```

File: cnf-tests/testsuites/e2esuite/fec/fec.go (hunk 1 above)
Comment (kononovn) [reply cnf-tests/testsuites/e2esuite/fec/fec.go:87]: Updated, please take a look one more time.

**Comment by zeeke** [comment]
/approve

Letting others to review this. 

Also, need to fix `ci/prow/ci`:
```
274629b1 cnf-tests: added fec isSecureBoot function
88b23bcf cnf tests: Fix fec secure boot test case
ERROR: must use authorized component (features/s2i/docs/depbot/ztp/vendor/infra/cnf-tests/oot-driver/owners/tools)
```

File: cnf-tests/testsuites/e2esuite/fec/fec.go [hunk 3]
```diff
@@ -3,12 +3,12 @@ package fec
 import (
 	"context"
 	"fmt"
-	"time"
-
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-
 	corev1 "k8s.io/api/core/v1"
+	"strings"
```
Comment (SchSeba) [inline cnf-tests/testsuites/e2esuite/fec/fec.go:9]: can you please move this one up  and split  like this

```
"context"
"fmt"
"strings"
"time"

. "github.com/onsi/ginkgo/v2"
. "github.com/onsi/gomega"

corev1 "k8s.io/api/core/v1"
k8s.io/apimachinery/pkg/api/errors"
"k8s.io/apimachinery/pkg/api/meta"
metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

**Review by SchSeba — COMMENTED** [review]
nice work just a small comment about the imports and if you can please update the commit messages to make the CI happy

**Comment by zeeke** [comment]
failing test cases in [ci/prow/e2e-aws-ci-tests](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift-kni_cnf-features-deploy/1766/pull-ci-openshift-kni-cnf-features-deploy-master-e2e-aws-ci-tests/1752036771073363968) are not related to thisPR :

```
Performance Test: [It] [rfe_id:27363][performance] CPU Management Verification of configuration on the worker node [test_id:37862][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Verify CPU affinity mask, CPU reservation and CPU isolation on worker node expand_more	3s
Performance Test: [It] [rfe_id:27350][performance]Topology Manager [test_id:26932][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] should be enabled with the policy specified in profile expand_more
```

/override ci/prow/e2e-aws-ci-tests

**Pull request merged**